app/views/admin/shelters/index.html.erb

```diff
@@ -7,4 +7,10 @@
     <%= link_to "Update #{shelter.name}", "/shelters/#{shelter.id}/edit" %>
     <%= link_to "Delete #{shelter.name}", "/shelters/#{shelter.id}", method: :delete %>
   </div>
+<% end %>
+
+<h3>Shelters with Pending Applications</h3>
+
+<% @pending_applications.each do |shelter| %>
```

Comment:
Perhaps this naming is a bit weird, but I like the way it reads, especially considering the wording of the US.